src/internal/handlers.ts

```diff
@@ -0,0 +1,99 @@
+import { LoadContext, navigateBack } from '../public';
+import { Communication, sendMessageEventToChild, sendMessageToParent } from './communication';
+
+class Handlers {
```

Comment:
I could change this to HandlersPrivate to be more consistent @sachingupta what do you think?